waspc/src/Wasp/Generator/NpmInstall.hs

```diff
@@ -181,27 +166,3 @@ installProjectNpmDependencies messagesChan projectDir =
           case J._data jobMsg of
             J.JobOutput {} -> printJobMessagePrefixed jobMsg >> processMessages chan
             J.JobExit {} -> return ()
-
-installWebAppAndServerNpmDependencies ::
```

Comment:
Since the project root `npm install` handles all of the subpackages we define, no need to run separate `npm install` for the subpackages.